website/static/js/project.js

```diff
@@ -241,6 +241,14 @@ $(document).ready(function() {
         trigger: 'hover'
     });
 
+    var adminInfoHtml = 'These users are not contributors on this component, ' +
+        'but can view it because they are administrators on a parent project.';
+
+    $('.admin-info').attr(
```

Comment:
In both cases, to preserve consistency with previous code (see the logic right above this in `project.js`. I didn't want to introduce multiple ways for adding popover text, but also didn't want to refactor existing popovers.